app/__tests__/app.test.js

```diff
@@ -227,4 +228,62 @@ describe(`GET:/api/users --sad path`, () => {
         expect(message).toBe("Invalid path");
       });
   });
-});
\ No newline at end of file
+});
+
+describe.only('GET /api/articles --happy path', () => {
+  test('200: responds with an an array of articles objects having eight properties', () => {
+    return request(app)
+    .get('/api/articles')
+    .expect(200)
+    .then(({body: {articles}}) => {
+      expect(articles.length).toBeGreaterThan(0);
+      expect(Array.isArray(articles)).toBe(true);
+      expect(articles).toBeSorted({ descending: true })
```

Comment:
you should specify _how_ these are sorted (should be by `created_at` by default)